### DISCUSSION TITLE
GitHub support!

### DISCUSSION THREAD

**Discussion by wemeasure** [community/community#32780] [New to GitHub] [answered]
**Hey everyone.  I really need help from GitHub support.  We upgraded to the pro version and have submitted a ticket to which we never got a response and at this point, don't know what to do.  The issue we're having is we're unable to add any secondary emails to have access to our account.  It either tells us that the email is already in use or goes through a loop and never allows them to be added to the account.  It tells me the email is already in use and if I try a different email, when they get the verification link, it leads them to a page to create a new GitHub account.  Please direct us to a way of reaching support and resolving this issue.  Thanks.**

**Comment by queenofcorgis** [comment, marked as answer]
Hi there - thank you so much for your patience, I took a look and can confirm your ticket was received. Unfortunately, the Community on Discussions team can't escalate tickets nor provide an estimate as it depends on the Support team's resources and the tickets ahead of yours. Appreciate you waiting - I will see if there are any other solutions for this in the meantime.

**Comment by queenofcorgis** [comment]
Chiming back in - some possible solutions while you wait. Is the email unverified on another account or are multiple people using the same account?

**Reply by nathaneneja** [reply]
Yeah it is true

**Comment by nathaneneja** [comment]
you know i am really good
